.github/workflows/deploy-develop.yml

```diff
@@ -3,14 +3,14 @@ on:
   push:
     branches:
       - develop
+      - bugfix/jon/1632-stop-checking-in-pdfs
 jobs:
   build-deploy:
     runs-on: ubuntu-latest
     steps:
       - name: Cleanup disk
         uses: curoky/cleanup-disk-action@v2.0
-        with:
-          retain: "python,node"
```

Comment:
We're going to reinstall these in a minute, but it doesn't hurt to remove the default versions.